cypress/integration/examples/add_conflicting_product.spec.js

```diff
@@ -0,0 +1,48 @@
+/// <reference types="cypress" />
+
+import Chance from "chance";
+const chance = new Chance();
+
+const randomCheckbox = chance.integer({ min: 0, max: 6 });
+
+describe("Choose new page", () => {
+  it("finds the link to the product page, clicks it and gets directed there", () => {
+    cy.visit("http://localhost:4000");
+    cy.wait(3000);
```

Comment:
This wait looks like a potential issue. Loading time depends on server load and environment, thus, you can not expect to have the same wait time for every device. 
But it should not be required to wait here at all.